src/real/realScript.sml

```diff
@@ -4291,4 +4291,12 @@ Proof
   >> match_mp_tac REAL_LE_LMUL_IMP >> fs[]
 QED
 
+Theorem REAL_DIV_ZERO:
+  !a b.
+    b <> 0 ==>
+    ((a / b = 0) <=> (a = 0))
+Proof
+  rpt strip_tac \\ EQ_TAC \\ fs[REAL_DIV_LZERO, real_div]
+QED
+
```

Comment:
I think you could make this an iff: `(a/b = 0) <=> (a = 0) \/ (b = 0)`

Though this then depends on the way `inv 0 = 0`, I think the added convenience is worth it.

Reply:
Done in the latest commit. I did not even need to change the proof script.